orion/controllers/announce/announceController.go

```diff
@@ -0,0 +1,115 @@
+package announce
+
+import (
+	"errors"
+	"net/http"
+	"regexp"
+	
+	"github.com/gin-gonic/gin"
+)
+
+const REGEX_ANNOUNCE_ID = `^[[:alnum:]]+(_[[:alnum:]]+)*$`
+const REGEX_TITLE = `^[A-Z0-9][[:alnum:]-]*([- _]([(]?#\d[)]?|&|([(]?[[:alnum:]]+[)]?)))*$`
```

Comment:
I appreciate this. For now, we can keep things simple and just allow non-empty strings for message, author, etc.